app/controllers/creatorships_controller.rb

```diff
@@ -17,12 +17,17 @@ def show
 
   # Update the selected creatorships.
   def update
-    @creatorships = @creatorships.where(id: params[:selected])
+    selected_ids = Array.wrap(params[:selected]).reject(&:blank?)
+    if selected_ids.blank?
+      flash[:caution] = t(".must_select_request")
```

Comment:
It would be simpler and less fragile to potential changes to always call `@creatorships.where(id: params[:selected])` and decide whether to show the message based on the return value of that, instead of this extra `Array` check